src/app/expansion-panel/expansion-panel.component.ts

```diff
@@ -8,12 +9,23 @@ import { ChangeDetectionStrategy, Component, EventEmitter, Output } from '@angul
   templateUrl: './expansion-panel.component.html',
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class ExpansionPanelComponent {
+export class ExpansionPanelComponent implements OnInit {
+  @Input() questionPanel?: boolean = false;
```

Comment:
```suggestion
  @Input() configOverride?: Partial<ExpansionPanelConfig>;
```
This would allow users of the component to customise as much as they want